src/plugins/dataflow/nodes/dataflow-node.scss

```diff
@@ -435,15 +435,175 @@ $node-inner-width: 140px;
     .decorated-inputs {
       width: $node-width;
     }
+    .node-bottom-section{
+      width: 100%;
+      display: flex;
 
-    .node-graph {
-      padding: 5px 5px 0 0;
-      margin: 10px 0;
-      width: $node-inner-width;
-      background-color: white;
+      .node-bottom-buttons{
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        width: 15px;
+
+        .scale-buttons{
+          width: 100%;
+          height: 20px;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          border-radius: 1px;
+          margin-left: -2px;
+
+          &.plus {
+            border-bottom: 1px solid;
+            font-size: 14px;
+            border-top-left-radius: 3px;
+            &:active {
+              color: white;
+            }
+          }
+          &.minus {
+            border-top: 0px;
+            font-size: 20px;
+            padding-bottom: 3px;
+            border-bottom-left-radius: 5px;
+            &:active {
+              color: white;
+            }
+          }
+
+          &.sensor {
+            border-color: $sensor-blue;
+            color: $sensor-blue;
+            &:hover {
+              background-color: $sensor-blue-outline;
+            }
+            &:active {
+              background-color: $sensor-blue-control;
+            }
+          }
+
+          &.number {
+            border-color: $number-blue;
+            color: $number-blue;
+            &:hover {
+              background-color: $number-blue-outline;
+            }
+            &:active {
+              background-color: $number-blue-control;
+            }
+          }
```

Comment:
I don't think you should deal with this now, but it would be nice to come up with a better solution for handling different colors. Maybe you can try to clean this up during the clean dead code PT story. Variable cards can have their colors changed, which could be a good model to follow for this.

Reply:
I'll add a screenshot of this to the "clean dead code PT story"

Reply:
Were you suggesting using an inline style on the .tsx side? 

Such as style={{style=`${nodeName}-blue`}}
Where nodeName could be "generator", "timer", "math", etc

Reply:
No, it's all done in `.scss`. If you want to see how it works, get the `quantity-playground` repo, then check `quantity-node.scss`, `mixins.scss`, and `vars.scss`. I'll also point out that to really make this worth it, we'd want to use it in all parts of the css where colors are specified conditionally. This is something Ethan set up.

Reply:
Okay I added yr comment to the Clean dead code ticket and will look into `quantity-playground` repo when I get to that ticket. Thanks for that info.